app/views/dashboard/show.html.erb

```diff
@@ -1,27 +1,5 @@
-<div class="govuk-grid-row govuk-!-margin-bottom-6">
-  <div class="govuk-grid-column-full">
-    <div class="blue-header">
-      <h1 class="govuk-heading-xl blue-header__title">Welcome <%= current_user.name %></h1>
-      <%= govuk_link_to "Edit name", edit_preferred_name_path, class: "blue-header__text", "data-test": "edit-name" %>
-    </div>
-
-    <% if current_user.early_career_teacher? %>
-      <%= govuk_link_to "Go to your module materials",
-        cip_path(current_user.early_career_teacher_profile.core_induction_programme),
-        button: true %>
-    <% end %>
-
-    <% if current_user.mentor? && current_user.mentor_profile.core_induction_programme %>
-      <%= govuk_link_to "Go to your mentee's module materials",
```

Comment:
We have a cypress test that uses that. I think the test needs to be updated to use the copy from the start button :P